spec/order_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'order'
```

Comment:
The tests here are not isolated from the Menu class.  I.e. Your `subject` is using a real instance of `Menu` rather than a double.  You're injecting and instance of `Menu` in your 'Order' class so you could, in the tests, inject a double instead.

Fancy making an update to that effect?